classes/app.rb

```diff
@@ -1,4 +1,3 @@
-require 'json'
 require_relative './item'
```

Comment:
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @brytebe,

Good job so far!
✔️ Events work as expected
✔️ Use of lifecycle methods
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._